src/sentry/models/activity.py

```diff
@@ -40,7 +40,13 @@
 
 
 class ActivityManager(BaseManager["Activity"]):
-    def get_activities_for_group(self, group: Group, num: int) -> Sequence[Activity]:
+    def get_activities_for_group(self, group: Group) -> Sequence[Activity]:
```

Comment:
This function is called by ([link to code search](https://github.com/search?q=repo%3Agetsentry%2Fsentry+get_activities_for_group+path%3A%2F%5Esrc%5C%2Fsentry%5C%2F%2F&type=code)):
* the group_activities API
* the group_details API
* calling [update_groups](https://github.com/getsentry/sentry/blob/57b2228f4726db26f999341d5ac2744bf8b25063/src/sentry/api/helpers/group_index/update.py#L170) is called by:
  * the project group index API
  * the organization group index API